lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -84,6 +88,8 @@ def __init__(
         -----
         .. versionadded:: 1.2.1
             The *sharez* parameter.
+        .. versionadded:: TBD
```

Comment:
I don't know that we do these annotations?

Reply:
Should I remove both then?

Reply:
Yes, please.

Reply:
Done, though for what it's worth there are still a couple dozen similar 'versionadded' notes elsewhere in the file.